src/pages/user/wallet.tsx

```diff
@@ -162,6 +162,7 @@ export default function UserWallet() {
 
       return updated;
     }, {});
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [transactions]);
```

Comment:
I think it would make sense to add `starterBuzzAmount` in the dependency array here
```suggestion
  }, [starterBuzzAmount, transactions]);
```